eopf.yml

```diff
@@ -0,0 +1,15 @@
+version: 1
+project:
+  open_access: true
+  license:
+    content: CC-BY-4.0
```

Comment:
The license of this repo is Apache 2.0: https://github.com/EOPF-Sample-Service/eopf-sample-notebooks?tab=Apache-2.0-1-ov-file#readme

By the way, why both CC-By-4.0 and MIT are listed?

Reply:
Ok. Sorry I forgot we used Apache and not MIT. The reason to have 2 licenses is to have one for the code and one for markdown/rendered pages e.g. the jupyter book is CC-By-4.0 but the code is Apache.